### ISSUE TITLE
Felinids like showers

### ISSUE THREAD

**Issue by Inari-Whitebear** [tgstation/tgstation#72298]
Whenever I go use a shower as felinid I get a mood buff. Aren't they supposed to dislike that?

## Reproduction:
* Play felinid
* Use a shower

**Comment by san7890** [comment]
can you point to a PR/wiki post/forums thing or anything that says this should be a mechanic in the game

**Comment by Inari-Whitebear** [comment]
Maybe I misread their negative "being splashed with water" trait? I just figured they wouldn't like standing under a shower, if they don't like being sprayed with it.

Or at least that's it on the wiki: "Being sprayed with water interrupts any action they were in the process of doing (think climbing a table) and gives them a negative moodlet."

In code their hydrophobia trait has a shower icon and "Felinids don't like getting soaked with water."
Well, standing under the shower sounds like getting soaked.

Though apparently a PR to add this was closed without real comment as to why (#50635). So I guess we'll just go with "felinids dislike being soaked, maybe, kinda, sometimes."
